lib/twilio_text.rb

```diff
@@ -0,0 +1,21 @@
+
+require 'twilio-ruby'
+require 'denv'
+Dotenv.load
+
+class SMS
+  def text_message
+    account_sid = ENV['TWILIO_ACCOUNT_SID']
+    auth_token = ENV['TWILIO_AUTH_TOKEN']
+
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
```

Comment:
You can shorten this to `@client = Twilio::REST::Client.new ENV['TWILIO_ACCOUNT_SID'], ENV['TWILIO_AUTH_TOKEN']` (if you want to). Although Rubocop might not like it as it will be quite a long line, it does shorten the method and prevents repetition.